source/_posts/To-monitor-and-restart-services-using-Monit.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Image should be optimized with Alt text

Reply:
I have used the asset_image which has the last component as title that is an alternative to alt_text

Reply:
Done